basics/keeping-up-to-date/upgrade.md

```diff
@@ -95,7 +95,7 @@ release content in the existing shop.
 
 ### Disable cache
 
-You may have activated a caching system (eg. memcache) on your shop. In that case, make sure to disable it in "Advanced Parameters" > "Performance". You can enable it again once the upgrade process is done.
+You may have activated a caching system (eg. memcache) on your shop. In that case, make sure to disable it in "Advanced Parameters" > "Performance". You can enable it again once the upgrade process is done. Please don't forget to also delete the var/cache/prod and var/cache/dev folders.
```

Comment:
I'm 50/50 on adding this line.

Some upgrades will be OK without deleting these folders.

But also there is, in theory, no risk in deleting these folders.

What do you think? @PrestaShop/prestashop-maintainers

Reply:
I would go for:

`You might want to...` instead of asking to do it, as you said @matks - but overall the better idea would be to just click the "Clear cache" button :-)

Reply:
Thank you @kpodemski 

@pkirk could you please update your contribution 😉